README.md

```diff
@@ -189,6 +189,12 @@ Report issues for this extension in the [ocds-extensions repository](https://git
 
 ## Changelog
 
+### Unreleased
+
+* Add `Criterion.relatedLots` field.
+
+This extension was originally discussed in <https://github.com/open-contracting/ocds-extensions/issues/76>.
```

Comment:
We usually only include this sentence to reference discussions that happened prior to the initial version of the extension. Discussions about subsequent changes are linked from the pull requests so we don't need to link them in the changelog.

```suggestion
```